src/Forms/Label.tsx

```diff
@@ -74,9 +80,16 @@ const StyledLabel = styled.label<StyledLabelProps>`
   grid-template-areas: ${({ labelPosition, isLabelVisible }) => (
     generateGrid(labelPosition, isLabelVisible)
   )};
-  margin: ${fromTheme('ws', 'small')};
+  margin: ${fromTheme('ws', 'small')}
   align-items: baseline;
   gap: ${({ theme }) => (theme.ws.xsmall) + ' ' + (theme.ws.xsmall)};
+  ${({ hideError }) => (hideError
+    ? `margin: 0px;
+       gap: 0px;
+       grid-template-rows: 1fr;
+      `
+    : null
```

Comment:
Since this `null` value is in a tagged template literal, it would be converted to the string "null."

![image](https://user-images.githubusercontent.com/29589689/96286609-a3194300-0fae-11eb-8232-8a1ecf4da258.png)

I think better here instead of `null` would be just an empty string "".